src/main/java/io/cdap/plugin/servicenow/apiclient/ServiceNowTableAPIClientImpl.java

```diff
@@ -362,7 +362,7 @@ public Schema fetchServiceNowTableSchema(String tableName, FailureCollector coll
     return null;
   }
 
-  private List<SchemaResponse> parseSchemaResponse(String responseBody) {
+  public List<SchemaResponse> parseSchemaResponse(String responseBody) {
```

Comment:
Are these access modifier changed only for testing purpose??
If yes, can we add the annotation `@VisibleForTesting`.
This comment applies to the whole PR.

Reply:
At this place, it is not changed due to test. That’s why not adding the annotation here, but added at other places.

Reply:
Why did we changed private->public if it was not testing purpose??

Reply:
It was public only as this method is being used in another packages as well. Somehow it is showing this change here. But it needs to be public.

Reply:
> It was public only as this method is being used in another packages as well

which package, can you point to the code where it's being used? Trying to understand why we needed to change access modifier as part of this PR

Reply:
It is used in (public void validateTable(String tableName, SourceValueType valueType, FailureCollector collector)), in ServiceNowBaseConfig class.

Reply:
Sorry, My bad. I was referring different parse method by mistake. Got it correct now and added the annotation there.